### PR TITLE
Issue #3885 - CRA Production Build

### PR DESCRIPTION
**Pull request by jmmadsen**
I have created a working production build for the new create-react-app ui implementation. To run the production build, navigate into the `cra-server` directory. Run `npm run build` if you have not yet created the static build files, or to overwrite your old ones. Then, run `npm run prod` to start the production server. Visiting https://localhost:8091/aaa/home will take you to login page and then the application.

@davidradl @grahamwallis

### CODE REVIEW

File: open-metadata-implementation/user-interfaces/presentation-server/cra-server/index.js [hunk 1]
```diff
@@ -55,11 +55,8 @@ if (env === 'development') {
   // organize routes in another file
   app.use('/', router);
 } else {
-  // use Web Server routing for production build
-
-  // app.use(express.static(`${__dirname}/client/build`));
+  app.use('/', router);
```
Comment (davidradl) [inline open-metadata-implementation/user-interfaces/presentation-server/cra-server/index.js:58]: I do not understand why " app.use('/', router);" is in the if and the else block. If it is needed in both then please move it to before the if.

**Comment by davidradl** [comment]
Hi @jmmadsen , I ran the npm run build and get error :
> presentation-server-client@0.1.0 build /Users/davidradley/egeria/open-metadata-implementation/user-interfaces/presentation-server/cra-client
> react-scripts build

sh: react-scripts: command not found

**Comment by davidradl** [comment]
@jmmadsen I tried to run npm run prod and get : 
npm ERR! missing script: prodI was in the cra-server folder - what am I missing?

**Review by davidradl — CHANGES_REQUESTED** [review]
As per the comments I could not run the command successfully.

**Comment by jmmadsen** [comment]
David, please navigate into the `cra-client` directory and run `npm install`. Then, go back to the `cra-server` directory and run the instructions listed above again.

I think this should fix your issue.

File: open-metadata-implementation/user-interfaces/presentation-server/cra-server/index.js (hunk 1 above)
Comment (jmmadsen) [reply open-metadata-implementation/user-interfaces/presentation-server/cra-server/index.js]: Good catch - I originally thought it needed only to be in development, then realized it was needed in both but did not fix the structure. Just fixed this and pushed now.

**Comment by davidradl** [comment]
@jmmadsen looks to be working on chrome on dev and production;. The checks seemed to have failed in the pipeline. I am not sure if this is due to the code or the pipeline; I suspect the pipeline. I will try again tomorrow after another master merge has gone in this could be kicked off again.

**Comment by davidradl** [comment]
@jmmadsen It worked  must have been a glitch in the pipeline - I have merged

**Pull request merged**